### PR TITLE
Adding options to install idrac-packages and "all" srvadmin-packages.

### PR DESCRIPTION
**Pull request by palsveningson**
The idrac-packages is nice to have installed for racadm-usage.
The "all"-option is a nice feature.

### CODE REVIEW

**Review by vide — CHANGES_REQUESTED** [review]
First of all, thanks for your contribution!
Can you remove any RPM mention from the README, since the module targets Debian too?

Thanks!

**Comment by palsveningson** [comment]
Hi! Good point, I replaced it with packages.

BR!

File: manifests/install.pp [hunk 1]
```diff
@@ -61,4 +67,19 @@
       }
     }
   }
+  
+  if ( str2bool("${::omsa::install_idac}")) {
```
Comment (vide) [inline manifests/install.pp]: There's a typo here

File: README.md [hunk 2]
```diff
@@ -99,6 +99,15 @@ As a last note, you should know that if you want to customize the SNMP installat
  * `force_install`
  Force OMSA installation even when $manufacturer is not Dell. Default: false
 
+ * `install_idrac`
+ Install the meta package-idrac package. Default: false
```
Comment (vide) [inline README.md]: I think `Install the idrac meta package` is better

**Review by vide — CHANGES_REQUESTED** [review]
Ok, thinking it twice (a part from the other small fixes I commented) I think you should make the `srvadmin-all` package installation mutually exclusive with the other packages, with the current code, dependencies will break if by mistake you have `install_all` and some other install flag activated

**Comment by palsveningson** [comment]
Hi! I did some changes. Hope this is what you wanted :)

BR

**Comment by vide** [comment]
Hello @palsveningson sorry for the delay. In the end I've implemented your proposal with #3 so I'm gonna close this PR and merge the other one. Please checks if it does what you needed!

**Pull request closed without merging**